lib/conversion.c

```diff
@@ -88,8 +88,7 @@ bool tpm2_convert_pubkey(TPM2B_PUBLIC *public, pubkey_format format, const char
         return tpm2_convert_pubkey_ssl(&public->publicArea, format, path);
     }
     else if (format == pubkey_format_tss) {
-        // save raw as is
-        return files_save_bytes_to_file(path, (UINT8 *)public, sizeof(*public));
+        return files_save_pubkey(public, path);
```

Comment:
files_save_public seems like a better name...

Reply:
Indeed.

Reply:
And probably makes sense to rename all the *_pubkey to *_public